### PR TITLE
bolt-ul and bolt-ol testing and documentation

### PR DESCRIPTION
**Pull request by adamszalapski**
## Jira

http://vjira2:8080/browse/BDS-948
http://vjira2:8080/browse/BDS-949

## Summary

Add JEST test and documentation for testing to bolt-ol and bolt-ul components.

## Details

Please check if it's correct because i don't know how to prepare `TESTING.md` document. I added some JEST tests and i don't know if this amount of test it's enough. 
I create to task in jira for time logging:
bolt-ol: http://vjira2:8080/browse/BDS-1001
bolt-ul: http://vjira2:8080/browse/BDS-1000

Please log your time in those tasks.

## How to test

Check updated/added files.

### CODE REVIEW

File: packages/components/lists/bolt-ul/__tests__/ul.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        'Answer questions authoritatively and concisely. Avoid cluttering discussions with noise.',
+      ],
+    });
+    expect(results.ok).toBe(true);
+    expect(results.html).toMatchSnapshot();
+  });
+
+  test('with nested <bolt-ol> list compiles', async () => {
+    const results = await renderString(`
+      {% include '@bolt-components-ul/ul.twig' with {
```
Comment (sghoweri) [inline packages/components/lists/bolt-ul/__tests__/ul.js]: @remydenton @mikemai2awesome @danielamorse just wanted to point out this particular chunk of code — @adamszalapski and I figured out how to do nested components + use Twig extension in our Jest tests! 😉

File: docs-site/src/pages/pattern-lab/_patterns/02-components/ordered-list/05-ordered-list.twig [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {% include "@bolt-components-ol/ol.twig" with {
-  "items" : [
+  attributes: {
+    class: "custom-class",
```
Comment (sghoweri) [inline docs-site/src/pages/pattern-lab/_patterns/02-components/ordered-list/05-ordered-list.twig]: Can we update these custom class names in our examples to be written out how we’d recommend someone in real life to add custom classes (ie. use BEMIT namespaced CSS classes (**without** the Bolt namespace — aka CSS classes with a `js-` or `c-` prefix)?

Comment (sghoweri) [reply docs-site/src/pages/pattern-lab/_patterns/02-components/ordered-list/05-ordered-list.twig]: https://csswizardry.com/2015/08/bemit-taking-the-bem-naming-convention-a-step-further/#namespaces

Comment (adamszalapski) [reply docs-site/src/pages/pattern-lab/_patterns/02-components/ordered-list/05-ordered-list.twig]: Thanks, @sghoweri to pointing me that. I updated code for `bolt-ol` and `bolt-ul` component and test too.

**Comment by adamszalapski** [comment]
I update code and right now is following the patter that @mikemai2awesome pointing earlier.

**Pull request merged**